tests/test_classifier/test_prcurve.py

```diff
@@ -263,55 +329,3 @@ def test_no_scoring_function(self):
         oz = PrecisionRecallCurve(FakeClassifier())
         with pytest.raises(ModelError, match="requires .* predict_proba or decision_function"):
             oz._get_y_scores(self.binary.X.train)
-
-    def test_custom_iso_f1_scores(self):
```

Comment:
@zjpoh Is there a reason these tests were removed? Was this just a merge conflict thing?

Reply:
Interesting.... I do not recall remove this.

Reply:
Probably it was the automatic merge - no worries we caught it!